cal-itp/benefits
# Refactor: `AuthProvider` to `ClaimsProvider`

Closes #2233 - see Acceptance Criteria on that ticket.

## Testing locally

1. Checkout the `main` branch
1. Make sure your `local_fixtures.json` (or whatever fixture you are using) has an `AuthProvider` and an `EligiblityVerifier` that has a foreign key reference to that auth provider
1. Run `bin/reset_db.sh` (now your local DB mirrors the structure of the `dev` environment)
1. Checkout this PR's branch
1. Run `bin/init.sh` (running this PR's migration on top of existing DB structure)

- [ ] For all `EligibilityVerifier`s that with a reference to an `AuthProvider`, confirm those references were maintained, only now the field is called "Claims provider" in the admin interface, `claims_provider` in the database
- [ ] Confirm the `Can X auth provider` permissions are removed from the database
- [ ] Confirm the `Can X claims provider` permissions are added to the database
- [ ] Confirm the Cal-ITP group has `Can [view|change] claims provider` permissions
- [ ] Flows using claims from IdG should still work
